clean-modular-springboot-webapp/src/main/resources/archetype-resources/__rootArtifactId__-server/src/main/java/Application.java

```diff
@@ -6,19 +6,32 @@
 import org.springframework.boot.web.servlet.ServletComponentScan;
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 
+import java.io.File;
+import java.util.Objects;
+
 @SpringBootApplication
 @ServletComponentScan
-public class Application
-    extends SpringBootServletInitializer {
-  
-  public static void main(String[] args) {
-    SpringApplication.run(Application.class,
-                          args);
-  }
-  
-  @Override
-  protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
-    return builder.sources(Application.class);
-  }
-  
+public class Application extends SpringBootServletInitializer {
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
+        return builder.sources(Application.class);
+    }
+
+    @Component
+    public static class EmbeddedServletContainerConfig implements WebServerFactoryCustomizer<TomcatServletWebServerFactory> {
+        @Override
+        public void customize(TomcatServletWebServerFactory factory) {
+            /**
+             * You have to set a document root here, otherwise RemoteServiceServlet will failed to find the
+             * corresponding serializationPolicyFilePath on a temporary web server started by spring boot application:
+             * servlet.getServletContext().getResourceAsStream(serializationPolicyFilePath) returns null.
+             * This has impact that java.io.Serializable can be no more used in RPC, only IsSerializable works.
+             * */
+            factory.setDocumentRoot(new File(Objects.requireNonNull(getClass().getResource("/")).getFile(), "launcherDir"));
+        }
+    }
```

Comment:
the clean module does not have a RPC service implemented. so I would like to avoid code that is not needed.

Reply:
In clean module people may also using RPC. I put extra comment for people not planning use it, it is better?

Reply:
I prefer removing the code.  It's the clean module and should only contain code really needed.